Content.Server/GameTicking/GameTicker.Player.cs

```diff
@@ -65,6 +68,20 @@ private async void PlayerStatusChanged(object? sender, SessionStatusEventArgs ar
 
                     DebugTools.AssertNotNull(data);
 
+                    // Persistence hook
+                    if (data!.Mind == null)
```

Comment:
Before sending the player to the lobby, check if there are any Actors in which the player was attached to on the last save.